DependencyInjection/Compiler/EntityListenerPass.php

```diff
@@ -47,6 +47,7 @@ public function process(ContainerBuilder $container)
                 }
 
                 $resolver = $container->findDefinition($resolverId);
+                $resolver->setPublic(true);
```

Comment:
would be better to use a ServiceLocator (by changing the lazy-loading listener resolver) rather than forcing to use public services

Reply:
Not sure what do you mean. I don't want to change behaviour. Tests expect this service to be public, meaning having it directly accessible from container. Making it otherwise would be BC break. I have done this change only to fix tests.